feature/search/presentation/src/main/java/com/sanaa/presentation/screen/componants/RecentSearchItem.kt

```diff
@@ -0,0 +1,78 @@
+package com.sanaa.presentation.screen.componants
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sanaa.designsystem.R
+import com.sanaa.designsystem.design_system.theme.Theme
+
+@Composable
+fun RecentSearchItem(
+    text: String = "Shutter island",
+    isLast: Boolean = false,
+    onCancelClicked: () -> Unit = {}
+) {
+    Column {
+        Row(
+            modifier = Modifier
+                .padding(horizontal = 16.dp)
+                .padding(bottom = 12.dp)
+                .fillMaxWidth(),
+            horizontalArrangement = Arrangement.spacedBy(8.dp),
+            verticalAlignment = Alignment.CenterVertically
+
+        ) {
+            Icon(
```

Comment:
You should add the size of the icon here 
```suggestion
            Icon(
                modifier = Modifier.size(20.dp),
```